osf/models/user_message.py

```diff
@@ -71,9 +71,9 @@ class UserMessage(BaseModel, ObjectIDMixin):
         on_delete=models.CASCADE,
         help_text='The institution associated with this message.'
     )
-    is_sender_CCed = models.BooleanField(
+    is_sender_BCCed = models.BooleanField(
         default=False,
-        help_text='The boolean value that indicates whether other institutional admins were CCed',
+        help_text='The boolean value that indicates whether other institutional admins were BCCed',
```

Comment:
This should probably say, "The boolean value that indicates whether the sender is BCCed"